### PR TITLE
Restore correct styles to text input

### PR DESCRIPTION
**Pull request by Jmuskett**
## What does this do?

Restore correct styling to text input component

### CODE REVIEW

File: .lintstagedrc [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {
   "*.+(js|ts|tsx)": [
     "eslint",
-    "jest --findRelatedTests",
-    "jest --passWithNoTests"
```
Comment (fleonard) [inline .lintstagedrc:5]: are we going to drop this?

File: src/TextInput/TextInput.tsx [hunk 2]
```diff
@@ -8,6 +9,7 @@ import { CommonFieldTypes } from 'Field/types/commonFieldTypes'
 
 interface Props extends CommonFieldTypes {
   type?: 'text' | 'email' | 'password' | 'time' | 'date'
+  className?: string
```
Comment (fleonard) [inline src/TextInput/TextInput.tsx:12]: isn't this already part of `CommonFieldTypes`?

File: .lintstagedrc (hunk 1 above)
Comment (Jmuskett) [reply .lintstagedrc]: unintentional - restored in [64a25](https://github.com/marshmallow-insurance/smores-react/pull/2045/commits/64a25440dceacadbe36831f6b74e921de6626fa0)

File: src/TextInput/TextInput.tsx (hunk 2 above)
Comment (Jmuskett) [reply src/TextInput/TextInput.tsx]: yup, my bad - fixed in [0c4b15](https://github.com/marshmallow-insurance/smores-react/pull/2045/commits/0c4b158dc1349d31f74ff8d1a31c7446eab400b0)

**Review by fleonard — APPROVED** [review]
Are you going to update the changelog in another pr?

**Review by fleonard — APPROVED** [review]

**Pull request merged**